onBoardProject/components/loading_icon.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import {ActivityIndicator} from 'react-native';
 
-const LoadingIcon = ({isIconAnimating}) => (
+const LoadingIcon = ({isIconAnimating}:any) => (
```

Comment:
Aqui seria bom vc tipar as props do componente:

```suggestion
export interface LoadingIconProps {
  isIconAnimating: boolean;
}

const LoadingIcon: React.FC<LoadingIconProps> = ({ isIconAnimating }) => (
```